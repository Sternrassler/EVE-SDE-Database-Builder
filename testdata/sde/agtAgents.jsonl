{"agentID":3008416,"agentTypeID":2,"divisionID":20,"corporationID":1000035,"locationID":60003760,"level":4,"quality":20,"agentName":"Katsen Isha","isLocator":0}
{"agentID":3008417,"agentTypeID":2,"divisionID":20,"corporationID":1000084,"locationID":60008494,"level":4,"quality":0,"agentName":"Baran Menthral","isLocator":0}
{"agentID":3008418,"agentTypeID":2,"divisionID":20,"corporationID":1000051,"locationID":60004588,"level":3,"quality":-20,"agentName":"Artan Firas","isLocator":1}
