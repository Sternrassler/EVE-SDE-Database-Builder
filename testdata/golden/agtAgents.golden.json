[
  {
    "agentID": 3008416,
    "divisionID": 20,
    "corporationID": 1000035,
    "locationID": 60003760,
    "level": 4,
    "quality": 20,
    "agentTypeID": 2,
    "isLocator": 0
  },
  {
    "agentID": 3008417,
    "divisionID": 20,
    "corporationID": 1000084,
    "locationID": 60008494,
    "level": 4,
    "quality": 0,
    "agentTypeID": 2,
    "isLocator": 0
  },
  {
    "agentID": 3008418,
    "divisionID": 20,
    "corporationID": 1000051,
    "locationID": 60004588,
    "level": 3,
    "quality": -20,
    "agentTypeID": 2,
    "isLocator": 1
  }
]